source/routing/next_stop_time.cpp

```diff
@@ -437,7 +437,7 @@ CachedNextStopTime CachedNextStopTimeManager::CacheCreator::operator()(const Cac
 
 CachedNextStopTime::DtStFromJpp::DtStFromJpp(const vDtStByJpp& map) {
     size_t s = 0;
-    for (const auto& elt: map) {
+    for (const auto elt: map) {
```

Comment:
Why? It yields something as `std::pair<&T, &U>` ?

Reply:
It uses those : https://github.com/CanalTP/utils/blob/master/idx_map.h#L130-L131